config/token.go

```diff
@@ -18,8 +18,36 @@ type TokenClaim struct {
 	jwt.StandardClaims
 }
 
+// HTTP status code 200 and tokenGenerated model in data
+// swagger:response tokenGenerated
+type swaggTokenGeneratedResp struct {
```

Comment:
I am starting to review. Do handlers need a refactor to add a wrapper for the response ?

Reply:
No, It's an addition for swagger documentation, No refactor need it

Reply:
pretty late here, but do you think is a way to avoid the fact of having to add a new type like this one for each API endpoint?

Reply:
yep, those definitions are important, sure thing :) My point is that we only have one piece of software that handles all the responses in an generic way. If there is no other option, perhaps we should create a folder where we place all this only-for-documentation code.
Let's see what they tell you :)

Reply:
Good idea +1